src/RegonValidator/RegonValidator.cs

```diff
@@ -10,6 +10,8 @@ public static class RegonValidator
 
         public static bool IsValid(string regon)
         {
+            var sonarQubeTrap = true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'sonarQubeTrap' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)